app/views/pages/api-reference/liquid/introduction.liquid

```diff
@@ -9,6 +9,8 @@ metadata:
 Liquid (or Liquid markup) is a template language used in platformOS to build dynamic pages, and to provide dynamic configuration (e.g. based on currently logged in user).
 Use Liquid to provide an authorization policy for forms and pages, or to specify notifications (email, SMS, API call). We have added a lot of  filters and tags to make your life easier.
 
+{% include 'alert/note', content: 'Liquid has a new syntax — visit the [documentation of the liquid tag]() for an example. We are updating all of our Liquid code examples to reflect this change. ' %}
```

Comment:
`()` should have a link inside to be point to a page, right now its not a valid link IMO

Reply:
Thank you! 
Fixed it right after the commit. https://documentation.platformos.com/api-reference/liquid/introduction